src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,49 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap-size;
+
+  padding: $field-paddings;
+  width: $calendar-field-width;
+
+  @for $i from 28 through 31 {
+    &--month-length--#{$i} {
+      .calendar__cell:nth-child(n + #{$i+1}) {
+        display: none;
+      }
+    }
+  }
+
+  @each $name, $number in $days {
+    &--start-day--#{$name} {
+      .calendar__cell:first-child {
+        margin-left: ($number - 1) * ($cell-size + $gap-size);
+      }
+    }
+  }
+
+  &__cell {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    @include cell($cell-size);
+
+    cursor: pointer;
+    background-color: $cell-color;
+    border: 1px solid $border-color;
+
+    transition: background-color 0.5s, transform 0.5s;
+
+    @for $i from 1 through 31 {
+      &:nth-child(n+#{$i})::after() {
```

Comment:
Remove the parentheses after `::after`. They are not necessary and may cause issues in some browsers.